src/counter.rs

```diff
@@ -571,38 +571,4 @@ mod tests {
         assert_eq!(local_vec_1.with_label_values(&["v1", "v2"]).get(), 0);
         assert_eq!(vec.with_label_values(&["v1", "v2"]).get(), 34);
     }
-
-    #[cfg(debug_assertions)]
-    #[test]
-    #[should_panic(expected = "assertion failed")]
-    fn test_counter_negative_inc() {
-        let counter = Counter::new("foo", "bar").unwrap();
-        counter.inc_by(-42.0);
-    }
-
-    #[cfg(debug_assertions)]
-    #[test]
-    #[should_panic(expected = "assertion failed")]
-    fn test_local_counter_negative_inc() {
-        let counter = Counter::new("foo", "bar").unwrap();
-        let local = counter.local();
-        local.inc_by(-42.0);
-    }
```

Comment:
I think only tests for `IntCounter` need to be removed. `Counter` accepts floats which can be negative.